packages/media-query/src/use-breakpoint.ts

```diff
@@ -25,20 +25,25 @@ export function useBreakpoint(defaultBreakpoint?: string) {
   )
 
```

Comment:
Don't we need to set the default value to `useBreakpoint` as well as `useBreakpointValue`?

Reply:
Your right, I think we should move the default value from useBreakpointValue to useBreakpoint to ensure a safe SSR+CSR for both hooks.